circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   xcode:
-    version: 8.2
+    version: 8.3
```

Comment:
Ah we have spotlight indexing disabled by default, good call on the `mdimport` @diogot.